docs/platform/what-is-tizen/versions/tizen-7-0-m2.md

```diff
@@ -0,0 +1,112 @@
+
+## Natural User Interface
+
+- **Mini Avatar UI Assistant(Consistent Interaction Through Voice)**:
+
+  Tizen provides UI assistance technology that allows users to control the UI through voice without additional processing in applications
+
+  The Mini avatar is displayed in a separate layer. And it enables more natural interaction by showing the appropriate feedback for the situation.
+  It also shows automatically generated commands for all controllable UI objects, and when the user says a command, it sends events to a selected object.
+
+  **Figure: Mini Avatar UI Assistant**
+
+  ![MiniAvartarUIAssistance](./media/7.0_MiniAvartarUIAssistance.png)
+
+- **3D User Interface**:
+
+  Tizen provides a unified rendering engine. All UI elements have depth information and they are located in 3D space
+
+  As all UI elements are rendered together, we can optimize the rendering pass and minimize memory usage.
+
+	The biggest advantage is that it reduces the complexity quite significantly for creating a 3D UI.
+	This is because almost all the features used in 2D UI applications such as event handling, text input, animations, and accessibility, etc., are still available in 3D
+
+  **Figure: 3D Scene Building**
+
+  ![3DSceneBuilding](./media/7.0_3DSceneBuilding.png)
+
+	**Figure: 3D Scene Navigation**
+
+  ![3DSceneNavigation](./media/7.0_3DSceneNavigation.png)
+
+
+## Platform Compatibility
+
+- **Tizen-Android HAL Adaptation**:
+
+  Lots of time and effort are used to develop hardware abstraction layers and Tizen has introduced HAL APIs in Tizen 6.5.
+	While we had some success with this, many SoC chip makers only support Android. To address this issue,
+	we introduced a new solution to run Tizen on Android supported SoC by providing an adaptation bridge to Android HAL
+	
+	To make Tizen run on Android HAL and Kernel, we added the open source project Libhybris and defined a new glue layer to access hardware devices through Android HAL.
+  Also, to solve compatibility issues between bionic in Android and glibc in Tizen, we used separate namespaces for the Android System and Hal container.
+  
+  **Figure: Tizen-Android Hal Adaptation**
+
+  ![TizenAndroidHalAdaptation](./media/7.0_TizenAndroidHalAdaptation.png)
+
+
+## Cross Platform Technology
+	
+[!NOTE]
+If you are already using popular cross-platform frameworks such as Web, Flutter or Maui, you can deploy your services to Tizen with minimal effort
+	
+- **New support for Tizen Web eco platform**:
+
+  Web is arguably the most widely used cross-platform technology. And more and more services are based on web on Tizen devices,
+  so it is business critical to support up-to-date web engines. To achieve that, we had to reduce time and effort to port a new version of the open source Chromium web engine.
+  A big move we took with Tizen 7.0 was re-architecturing our web engine porting layer. Instead of using our thick Tizen platform port,
```

Comment:
```suggestion
  A big move we took with Tizen 7.0 was re-architecting our web engine porting layer. Instead of using our thick Tizen platform port,
```